portal/src/components/FilterCategories/FilterDropdown.vue

```diff
@@ -2,7 +2,7 @@
   <div class="dropdown-container">
     <label class="dropdown-container__label">{{ label }}:</label>
 
-    <div class="dropdown-container__select" @click="onToggle">
+    <div class="dropdown-container__select" :data-test="'filter_' + field" @click="onToggle">
```

Comment:
Would call this attribute data-field or similar. No reason to limit the use to the tests